### PR TITLE
Use selenium download

### PR DESCRIPTION
**Pull request by ellenmobify**
Status: **Ready for Review** 
Owner: 
Reviewers:  @marlowpayne @mikenikles

## Changes
- Latest version of Chrome (54) is too new for our outdated version of Chromedriver. This PR switches over to using https://github.com/groupon/selenium-download to manage the downloading of the latest versions of Selenium server and Chromedriver, with some moving/renaming to keep the same file structure as previous versions of nightwatch-commands

## Todos:
- [x] Passed linting check (run `grunt lint`)
- [x] Updated README
- [x] Updated CHANGELOG

### Feedback:
_none so far_

## How To Test
- Checkout this branch
- `npm install`
- `npm link`
- `grunt lint`
-  In `package.json`, change the nightwatch-commands line to `"nightwatch-commands": "mobify/nightwatch-commands#use-selenium-download",`, then run the end-to-end tests. You will need Chrome 54 on your machine to test locally.
- To test on CircleCI, update `circle.yml` to use node 4 and the latest stable version of Chrome. See  machine and dependencies https://github.com/mobify/lancome-progressive/blob/automation_construction/circle.yml
- Or just check out https://github.com/mobify/lancome-progressive/tree/automation_construction

### CODE REVIEW

**Comment by marlowpayne** [comment]
Have we investigated/tested how backwards-compatible these changes are?

**Comment by ellenmobify** [comment]
This change will require Node 4 and Chrome 54. For most projects this means updating your local dev environment and circle.yml.

Adaptive 1.x https://github.com/mobify/carnival-cruises-mobile/tree/test-nightwatch-commands
Adaptive 2.x https://github.com/mobify/holland-america-mobile/tree/test-nightwatch-commands

**Comment by marlowpayne** [comment]
So previously I believe `nightwatch-commands` was fully compatible with Node v0.X.X. Now that v4 is a hard requirement (tests fail with Node v0), we might want to consider a major version bump for this change.

File: package.json [hunk 1]
```diff
@@ -11,24 +11,22 @@
...
   },
   "homepage": "https://github.com/mobify/nightwatch-commands",
   "dependencies": {
-    "mkdirp": "^0.5.1",
     "async": "^0.2.10",
     "chalk": "^0.4.0",
-    "mkdirp": "^0.5.0",
-    "unzip": "^0.1.9",
-    "rimraf": "^2.2.8"
+    "mkdirp": "^0.5.1",
```
Comment (marlowpayne) [inline package.json]: It would be a nice-to-have to get _all_ dependency versions pinned down with this update.

File: CHANGELOG [hunk 2]
```diff
@@ -1,3 +1,5 @@
+2.0.0
+    - Use [selenium-download](https://github.com/groupon/selenium-download) to manage downloading of Selenium server and Chromedriver. This requires node 4+ and Chrome 54 or higher.
```
Comment (marlowpayne) [inline CHANGELOG]: (Nice-to-have) Move the breaking change ("This requires node 4+ and Chrome 54 or higher.") into a separate line with the heading "BREAKING CHANGES", like we do on [Progressive](https://github.com/mobify/progressive-web-sdk/blob/develop/CHANGELOG.md).

**Pull request merged**